apps/aechannel/src/aesc_offchain_state.erl

```diff
@@ -132,14 +135,14 @@ check_reestablish_tx(SignedTx, State) ->
 check_update_tx(SignedTx, State, OnChainTrees, OnChainEnv, Opts) ->
     check_update_tx_(none, SignedTx, State, OnChainTrees, OnChainEnv, Opts).
 
-check_update_tx_(F, SignedTx, #state{signed_txs = Txs}=State, OnChainTrees,
-                 OnChainEnv, Opts) ->
+check_update_tx_(F, SignedTx, #state{signed_tx = OldSignedTx}=State, OnChainTrees,
```

Comment:
No function clause that matches ?NO_TX?

Reply:
Yes. All updates are based on preceding state. If there is no state - no update can be applied.